slsim/lens.py

```diff
@@ -339,29 +363,67 @@ def deflector_velocity_dispersion(self):
         """
         return self.deflector.velocity_dispersion(cosmo=self.cosmo)
 
+    @property
     def los_linear_distortions(self):
+        if self._los_linear_distortions_cache is None:
+            self._los_linear_distortions_cache = (
+                self._calculate_los_linear_distortions()
+            )
+        return self._los_linear_distortions_cache
+
+    def _calculate_los_linear_distortions(self):
```

Comment:
this function might better fit in the LOSConfig (or in another LOS related class that compiles and manages the different settings. In this way, we keep the Lens() class as clean as possible